db/migrate/20240726141600_create_backup_codes.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateBackupCodes < Sagittarius::Database::Migration[1.0]
+  def change
+    create_table :backup_codes do |t|
+      t.text :token, limit: 10
+      t.references :user, null: false, foreign_key: true
```

Comment:
If it does not cascade, we wouldn't be able to delete users with backup codes anymore.

Also, this column does not need it's index because it is covered in the index that we create manually below.
```suggestion
      t.references :user, null: false, foreign_key: { on_delete: :cascade }, index: false
```